lmcache/v1/storage_backend/connector/mooncakestore_connector.py

```diff
@@ -74,6 +75,25 @@ def load_from_env() -> "MooncakeStoreConfig":
             )
         return MooncakeStoreConfig.from_file(config_file_path)
 
+    @staticmethod
+    def load_from_lmcache_config(
+        config: "LMCacheEngineConfig",
+    ) -> "MooncakeStoreConfig":
+        """Load config from a file specified in the environment variable."""
+        extra_config = config.extra_config
+        if extra_config is None:
+            raise ValueError("The extra config is not set.")
+        return MooncakeStoreConfig(
+            local_hostname=str(extra_config["local_hostname"]),
```

Comment:
Are these arguments all necessary? If some of them is optional, maybe we can supply the default value if absent?

Reply:
Sure.